package.json

```diff
@@ -139,7 +143,7 @@
     "through2": "^4.0.2",
     "vinyl-source-stream": "^2.0.0",
     "vue-loader": "^15.9.8",
-    "vue-template-compiler": "^2.6.12",
+    "vue-template-compiler": "^2.6.14",
```

Comment:
same here, don't see the reason to bump with this pr

Reply:
see above